internal/config/queue.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+// QueueConfig queue configuration
+type QueueConfig struct {
+	//SaveASDRequestQueue represents a queue name to send processed ApiSpecDoc
+	SaveASDRequestQueue string `default:"storage-update-asd" envconfig:"SCRAPING_RESULT_QUEUE"`
```

Comment:
here this "SCRAPING_RESULT_QUEUE" 
have to be renamed to 
"SAVE_ASD_REQUEST_QUEUE" to provide correct environment variable name